packages/mosaic/sql/src/functions/list.ts

```diff
@@ -0,0 +1,58 @@
+import type {ExprValue, MaybeArray} from '../types.js';
+import { asLiteral } from '../util/ast.js';
+import { argsList, fn } from '../util/function.js';
+import { ListNode } from "../ast/list.js";
+import {ExprNode} from "../ast/node.js";
+
+function listFn(name: string, expr: MaybeArray<ExprValue>, ...args: unknown[]) {
+  return fn(name, expr, ...(argsList(args).map(asLiteral)));
+}
+
+/**
+ * Return a SQL AST node for a literal value. The supported types are
```

Comment:
This comment needs to be updated as it looks like it was copy and pasted and does not reflect changes